ui/package-lock.json

```diff
@@ -2877,15 +2877,15 @@
       }
     },
     "node_modules/@material-ui/core": {
-      "version": "4.11.4",
-      "resolved": "https://registry.npmjs.org/@material-ui/core/-/core-4.11.4.tgz",
-      "integrity": "sha512-oqb+lJ2Dl9HXI9orc6/aN8ZIAMkeThufA5iZELf2LQeBn2NtjVilF5D2w7e9RpntAzDb4jK5DsVhkfOvFY/8fg==",
+      "version": "4.12.4",
```

Comment:
Is dependency version bumping usually carried out as a separate piece of work? There are quite a lot of version bumps here